src/table/README.md

```diff
@@ -5,6 +5,7 @@ A table component based on fixed-data-table
 ## Props
 |Name|Type|Default|Description|
 |----|----|-------|-----------|
+| **touchScrollEnabled** | <code>Boolean</code> | <code>false</code> | *optional*. Enable touch scroll |
```

Comment:
`table` is actually the old (deprecated) `fixed-data-table` component, you should revert this edit

(this component will soon be removed from brc)